Sample Data/Custom/CiscoMerakiNativePoller_CL.json

```diff
@@ -0,0 +1,52 @@
+[
```

Comment:
I assume those are the values at the original. I think that sample data is more useful for testing as it is in the workspace. The reason is that we can ingest them using a standard CLv2 importer. Tagging @anki-narravula and @ashishsyal.

Reply:
For other Sample Data items, we are trying to gather data from the raw JSON so as to get the original data. In this case specifically, since we are using ingestion level parser, I can add the parsed data in the custom table. Please advise

Reply:
I think it makes sense to add the parsed data. I will discuss with Ashish and Sapan as well because I think that for content testing the parser info is more useful.